tests/test_capture_csv_rides.py

```diff
@@ -0,0 +1,28 @@
+import json
+import subprocess
+
+
+def test_basic(request, snapshot):
```

Comment:
I think we need some sort of README on how to run these tests. I'll be honest and say that my initial impression was to wonder why these tests aren't in rust, since there's [pretty good support](https://rust-cli.github.io/book/tutorial/testing.html#testing-cli-applications-by-running-them) for testing CLIs. I'm fine with keeping the python tests, though I'd appreciate a doc on setup requirements.

Reply:
Yes, I've added a section into `tests/README.md`.

In my mind this is pretty experimental right now.
A major piece of context is that we need a blessed path for writing acceptance tests for connectors, especially python sass connectors, that are as high leverage and "easy" as possible for the connector author -- and to that end, _must_ be within the Python ecosystem. Unless we come up with something better, I think this is that path.